lib/filter.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
 
-var CleanCSS = require('clean-css');
-var minimatch = require('minimatch');
-var Promise = require('bluebird');
+const CleanCSS = require('clean-css');
+const minimatch = require('minimatch');
+const Promise = require('bluebird');
 
 module.exports = function(str, data) {
-  var options = this.config.clean_css;
-  var path = data.path;
-  var exclude = options.exclude;
+  const options = this.config.clean_css;
+  const path = data.path;
+  let exclude = options.exclude;
   if (exclude && !Array.isArray(exclude)) exclude = [exclude];
 
   if (path && exclude && exclude.length) {
```

Comment:
I suggest using "Array.prototype.some()".

Reply:
actually I'm planning to refactor to micromatch after this PR (otherwise there will be merge conflict). micromatch can parse `exclude` in both string and array format, so L11 & L14 (after this PR) are no longer necessary.